client/src/components/SideBar/SideBar.js

```diff
@@ -24,18 +25,33 @@ export default {
 			this.$router.push({ name: 'Home' });
 		},
 		async addNewroom(newname) {
-			if (this.newRoom != newname) {
-				this.Lists.push(newname);
-			}
 			try {
 				let response = await axios.post(baseUrl + '/room', {
 					roomName: newname,
 				});
-				console.log(response);
+				if (this.newRoom != newname) {
```

Comment:
- Đoạn này em check roomName không bị trùng nhưng chỉ check được room tạo ngay trước nó, không check đc các room đã có trong DB
- Em phải kiểm tra newname (Phải đặt là newName) có trùng với bất kì room nào trong ```roomLists``` của em hay không

Reply:
Ngoài ra, lúc nào thằng this.newRoom của em cũng rỗng
Anh k thấy em gán giá trị cho nó khi nào cả